_posts/2021-01-23-modeling-ui-state.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: Modeling UI State on Android
```

Comment:
Seems that Modeling holds to "l" -> Modelling

Reply:
> Whether you’re modelling or modeling, you’re doing the same thing. The only difference is in the spelling—the one with the single L is preferred in the United States, while the one with two Ls is preferred everywhere else.

apparently both are [correct](https://www.grammarly.com/blog/modeling-or-modelling/)

I was surprised spell checked didn't catch it